applications/ShapeOptimizationApplication/python_scripts/design_logger_gid.py

```diff
@@ -64,25 +63,12 @@ def __DetermineOutputMode( self ):
         else:
             raise NameError("The following design output mode is not defined within a GiD output (name may be misspelled): " + output_mode)
 
-    # --------------------------------------------------------------------------
-    def __CreateGiDIO( self ):
-
-        gid_config = self.output_settings["output_format"]["gid_parameters"]
-        results_directory = self.output_settings["output_directory"].GetString()
-        design_history_file_path =  results_directory+"/"+self.design_history_filename
-
-        if self.write_design_surface:
-            self.gid_io = GiDOutputProcess(self.design_surface, design_history_file_path, gid_config)
-        elif self.write_optimization_model_part:
-            self.gid_io = GiDOutputProcess(self.optimization_model_part, design_history_file_path, gid_config)
-
     # --------------------------------------------------------------------------
     def __ModifySettingsToMatchDefaultGiDOutputProcess( self ):
         gid_parameters = self.output_settings["output_format"]["gid_parameters"]
 
         # Add nodal results
-        nodal_results = self.output_settings["nodal_results"]
-        self.output_settings["output_format"]["gid_parameters"]["result_file_configuration"].AddValue("nodal_results", nodal_results)
```

Comment:
@RiccardoRossi @loumalouomega please check out this specific change. @dbaumgaertner was using `AddValue` with an entry that already exists. Rapidjson used to accept it and replace the values, while the new json library skips it (it finds that an entry with that name already exists, and just exits without changing anything). Is this a known Behavior Change?

Reply:
That part of the code in particular was implemented originally by @RiccardoRossi fortunetely can be easily solved

Reply:
Done

Reply:
Not saying it is a bug, just that neither @dbaumgaertner nor me where aware of the change. In any case, thanks for the quick patch :+1: